src/spell.js

```diff
@@ -463,7 +465,7 @@ function scopeDeletedAt(spell) {
   for (const condition of whereConditions) {
     let found = false;
     walkExpr(condition, ({ type, value }) => {
-      if (type == 'id' && value == deletedAt) {
+      if (type === 'id' && value == deletedAt) {
```

Comment:
```
value === deletedAt
```

Reply:
if value is `null` or `undefined`, use `===` will return `false`.